src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="submit"
+      class="form"
+    >
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input_big"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input_small"
+      />
+    </form>
```

Comment:
Your form lacks a submit button. Even though it's not explicitly stated in the task, it's a good practice to include one in your form. Without it, users won't be able to submit the form.